docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -19,6 +19,10 @@ worker outage or other fault during query execution.
     example, Trino does not spend resources retrying a query that fails because
     its SQL cannot be parsed.
 
+    For a step-by-step guide explaining how to configure a Trino cluster with
```

Comment:
Should we reword to avoid the "see ... " and just use a purely descriptive link?

Reply:
Also ..

query resiliency or query processing resilience

Reply:
I think these sorts of links are better with a "Here's why you, the reader of this content, should care about this other content" introduction.